package.json

```diff
@@ -78,6 +78,7 @@
   },
   "dependencies": {
     "@material-ui/styles": "^4.9.0",
+    "@material-ui/utils": "^4.10.2",
```

Comment:
I don't think this is used? or did i miss it

Reply:
Yeah it's not used, not sure when I added it :\